### DISCUSSION TITLE
how to find and extract the contents of <flet.dropdown.Option object at 0x7f30c3f41030>

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#974] [Q&A]
I'm curious about how to view something or extract from this widget <flet.dropdown.Option object at 0x7f30c3f41030>

is there a way to do a dump or see the children that are in it ?

but it applies to other widgets, for example row , column , radio. Is there a way to see it

**Comment by ghost** [comment]
asking the same since i found no way to customize the style of drodown options.

**Comment by ndonkoHenri** [comment]
Please paste a sample code.

**Reply by ghost** [reply]
anything[ of the examples](https://flet.dev/docs/controls/dropdown) will do
the ony thing that can be customized is the text  and key value of flet.dropdown.Option().  bgcolor, border, width, height, etc. cannot be called, which breaks any custom design pattern. 

in addition it would be nice if one could put other controls inside the options, e.g. to display icons or buttons.
